src/main/java/com/icthh/xm/ms/entity/domain/serializer/SimpleLinkSerializer.java

```diff
@@ -39,7 +39,7 @@ protected void serializeObject(Link value, JsonGenerator jsonGenerator, Serializ
         writeInstant(jsonGenerator, provider, "endDate", value.getEndDate());
 
         jsonGenerator.writeObjectFieldStart("target");
-        if (value.getTarget() != null) {
+        if (Hibernate.isInitialized(value.getTarget()) && value.getTarget() != null) {
```

Comment:
let's removes this, it's better to fail with error

Reply:
done